package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "repository": "https://github.com/navikt/veilarbpersonflatefs",
   "scripts": {
-    "start": "craco start",
+    "start": "cross-env REACT_APP_MOCK=true craco start",
     "build": "craco build",
```

Comment:
Faktisk så må REACT_APP_MOCK=false på build scriptet for at "mock" sidene ikke skal være bygd med frontend appen

Reply:
Dessverre så vil importen i index.tsx sørge for at sidene ikke blir fjernet, men de andre mock greiene vil bli fjernet hvis man setter REACT_APP_MOCK=false. Godt sett.

`import { SpaMockContent } from './components/spa-mock/spa-mock';`

Reply:
Hvis avhengighetere blir importert `slikt require('./mock');` Så vil de da ikke bli bygd med :)